convert2rhel/cert.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright(C) 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import os
+import shutil
+import sys
+
+from convert2rhel.systeminfo import system_info
+from convert2rhel.toolopts import tool_opts
+from convert2rhel import utils
+
+
+class SystemCert(object):
+    _system_cert_dir = "/etc/pki/product-default/"
+
+    def __init__(self):
+        self._cert_path = self._get_cert_path()
+
+    @staticmethod
+    def _get_cert_path():
+        cert_dir = os.path.join(utils.DATA_DIR, system_info.arch, "rhel-certs", tool_opts.variant)
+        if not os.access(cert_dir, os.R_OK | os.W_OK):
+            sys.exit("Error: Could not access {0}.".format(cert_dir))
+        pem_filename = None
+        for filename in os.listdir(cert_dir):
+            if filename.endswith(".pem"):
+                pem_filename = filename
+                break
+        if not pem_filename:
```

Comment:
```suggestion
pem_filename = next(glob.iglob("*.pem"), None):
if not pem_filename:
```
or just to show wallrus operator:))),
```python
if not (pem_filename := next(glob.iglob("*.pem"), None)):
```

Reply:
It's only a py3.8 feature, won't have it in py2.6

Reply:
@SpyTec yes, I know, I just shared it because this was cool example where this walrus actually can help:) In the `suggested change` the version which is working on py26 as well

Reply:
I do like the one-liner, not too complex. How would you add `cert_dir` into the mix? `iglob(cert_dir, "*.pem")`?

Reply:
@SpyTec it is just the same like globbing syntax in bash (https://docs.python.org/3/library/glob.html#glob.glob), so just iglob("/some_path_to_cert_dir/*.pem"). You could also use os path join:
```python
In [5]: os.path.join(cert_dir, "sdsdsd/*.pem")
Out[5]: '/cert/dir/path/sdsdsd/*.pem'
```